### PR TITLE
Updated and added to potential validation section

### PR DESCRIPTION
**Pull request by RaduV-Dev**
Added two new lines, added to an existing line and modified the XML file size limit.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md [hunk 1]
```diff
@@ -428,9 +428,13 @@ When you upload your rule package XML file, the system validates the XML and che
...
 
-- The XML structure does not require formatting characters such as Spaces, Tabs or Carriage Return / Linefeed entries.  Take note of this when optimizing for space on uploads.
+- Ensure each Entity element contains a recommendedConfidence attribute.
+
+- When using the PowerShell Cmdlet there is a maximum return size of the Deserialized Data of approximately 1 megabyte.   This will affect the size of your rule pack XML file. Keep the uploaded file limited to a 770 kilobyte maximum as a suggested limit for consistent results without error when processing.
+
+- The XML structure does not require formatting characters such as Spaces, Tabs or Carriage Return / Linefeed entries.  Take note of this when optimizing for space on uploads. Tools such as Microsoft Visual Code provide join line features to compact the XML file.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md]:
```suggestion
- The XML structure does not require formatting characters such as spaces, tabs, or carriage return/linefeed entries.  Take note of this when optimizing for space on uploads. Tools such as Microsoft Visual Code provide join line features to compact the XML file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@RaduV-Dev

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**